deepchecks/vision/checks/distribution/new_labels.py

```diff
@@ -0,0 +1,204 @@
+            **kwargs
+    ):
+        super().__init__(**kwargs)
+        # validate input parameters:
+        if not isinstance(max_images_to_display_per_label, int):
+            raise DeepchecksValueError('max_num_images_to_display_per_label must be an integer')
+        if not isinstance(max_new_labels_to_display, int):
+            raise DeepchecksValueError('max_num_new_labels_to_display must be an integer')
+
+        self.max_images_to_display_per_label = max_images_to_display_per_label
+        self.max_new_labels_to_display = max_new_labels_to_display
+        self._class_id_counter = defaultdict(list)
+
+    def update(self, context: Context, batch: Batch, dataset_kind):
+        """Count number of appearances for each class_id in train and test."""
+        data = context.get_data_by_kind(dataset_kind)
+        classes_in_batch = data.get_classes(batch.labels)
+        for labels_per_image in classes_in_batch:
+            for label in labels_per_image:
+                if label not in self._class_id_counter:
+                    self._class_id_counter[label] = [0, 0]
+                self._class_id_counter[label][dataset_kind == DatasetKind.TEST] += 1
+
+    def compute(self, context: Context) -> CheckResult:
+        """Calculate which class_id are only available in the test data set and display them.
+
+        Returns
+        -------
+        CheckResult
+            value: A dictionary showing new class_ids introduced in the test set and number of times they were spotted.
+            display: Images containing said class_ids from the test set.
+        """
+        data = context.get_data_by_kind(DatasetKind.TEST)
+        class_id_only_in_test = defaultdict(list)
+        for class_id, value in self._class_id_counter.items():
+            if value[0] == 0:
+                class_id_only_in_test[class_id] = value[1]
+        class_id_only_in_test = dict(sorted(class_id_only_in_test.items(), key=lambda item: -item[1]))
+
+        # Create display
+        display = []
+        for class_id, num_occurrences in class_id_only_in_test.items():
+            # Create id of alphabetic characters
+            sid = ''.join([choice(string.ascii_uppercase) for _ in range(3)])
+            images_of_class_id = list(set(data.classes_indices[class_id]))[:self.max_images_to_display_per_label]
+            images_combine = ''.join([f'<div class="{sid}-item">{draw_image(data, x, class_id)}</div>'
+                                      for x in images_of_class_id])
+
+            html = HTML_TEMPLATE.format(
+                label_name=data.label_id_to_name(class_id),
+                images=images_combine,
+                count=format_number(num_occurrences),
+                id=sid
+            )
+            display.append(html)
+        class_id_only_in_test['all_labels'] = sum(data.n_of_samples_per_class.values())
+        return CheckResult(class_id_only_in_test, display=''.join(display[:self.max_new_labels_to_display]))
```

Comment:
display can also be a list of display objects, can do:
```suggestion
        return CheckResult(class_id_only_in_test, display=display[:self.max_new_labels_to_display])
```